src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_base.cpp

```diff
@@ -83,14 +83,20 @@ JitConstants ConvolutionKernelBase::GetJitConstants(const convolution_params& pa
         }
     }
 
+    return mem_consts;
+}
+
+JitConstants ConvolutionKernelBase::GetJitConstantsWithLoopUnroll(const convolution_params& params, const DispatchData& dispatchData) const {
```

Comment:
* could you explain why you introduced this new function?
* What is the intention of the behavior change? Do you intend to prevent unroll for a specific kernel? Is it OK in terms of performance/load time?

Reply:
LoopUnroll is need for only 3 conv kernels. But previously, this is done for all conv kernels.